logentries/recipes/default.rb

```diff
@@ -7,20 +7,13 @@
 # All rights reserved - Do Not Redistribute
 #
 
-cookbook_file '/etc/ssl/certs/logentries.all.crt' do
-  source 'logentries.all.crt'
-  owner 'root'
-  group 'root'
-  mode '0755'
-end
-
 logentries = node[:deploy].map{ |e| e[1][:logentries] }.compact.first
 
 template '/etc/rsyslog.d/logentries.conf' do
   source 'logentries.conf.erb'
   owner 'root'
   group 'root'
-  mode '0755'
+  mode '0644'
```

Comment:
Not related to the issue but I found this permission is not correct. I don't think this conf file needs x permission